friend-face-react/src/components/posts/CommentItem.js

```diff
@@ -14,21 +14,24 @@ function CommentItem(props) {
     <li className={classes.item}>
       <Card>
         <div className={classes.content}>
+          <div
+            className={classes.avatarimage}
+            style={{ backgroundColor: props.avatar }}
+          />
           <h3>{props.author}</h3>
           <h4>{props.date}</h4>
-          <h4>{props.likes}</h4>
-          <p>{props.content}</p>
+          
         </div>
+
         <div className={classes.actions}>
-          <button onClick={() =>LikeHandler(props)}>Like</button>
-          <button onClick={() =>DeleteHandler(props,setDel)}>Delete</button>
-          
+          <p>{props.content}</p>
+          <h4>Likes:  {props.likes}</h4>
+          <button onClick={() => LikeHandler(props)}>Like</button>
+          <button onClick={() => DeleteHandler(props, setDel)}>Delete</button>
```

Comment:
I'm a little unsure what the `setDel` is about - can you explain it, please?

Reply:
I was hoping changing the state would cause a re-render. Looking into that today.